src/requirements.md

```diff
@@ -38,8 +36,8 @@ Note: The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “
 - The client MUST be able to subscribe/unsubscribe to/from parachains.
 - The client MUST be able to send a new transaction.
 - The client MUST be able subscribe a new transaction's updates.
-- The client MUST support the following minimum subset of [RPC methods](https://docs.5ire.org/docs/substrate/rpc/):
-  - `childstate_GetStorage`, `state_getStorage` and `state_getReadProof`.
+- The client MUST support the following minimum subset of [RPC methods](https://paritytech.github.io/json-rpc-interface-spec/):
```

Comment:
we're breaking the interface requirement here that "Either all functions in a group are supported, or none of them" (at https://paritytech.github.io/json-rpc-interface-spec/grouping-functions-and-node-capabilities.html ), but I suppose that's something w3f will be happy to correct...

Reply:
also, I forgot, we should support `rpc_methods` (required by the same spec page)

Reply:
Added `rpc_methods`, other was left as is. If w3f will correct it possibly we will have to change the milestones